src/lib/cmdk/components/CommandList.svelte

```diff
@@ -56,11 +57,15 @@
 </script>
 
 {#if asChild}
-	<slot {list} {sizer} />
+	{#key $state.search === ''}
```

Comment:
I originally had this wrapping the entire component but that was causing a flicker in animations, so I've instead duplicated it above the default slot.

I think there's a risk this will cause unwanted animations on the items inside the list though 😕